packages/components/src/utils/frameProps.tsx

```diff
@@ -63,6 +73,7 @@ export type FrameProps = {
     flex?: Flex;
     position?: Position;
     cursor?: Cursor;
+    zIndex?: number;
```

Comment:
shouldn't we allow only numbers defined in `zIndices`? But it's probably too cruel, right?